specification/components/schemas/CareConnect-Task.yaml

```diff
@@ -72,11 +72,13 @@ properties:
                   type: string
```

Comment:
Noticed in the documentation that the "title: RecordReviewOutcome" which appears in the schema.

Doesn't seem right. Maybe should be AvailableActionsForUser?